proto/stream_service.proto

```diff
@@ -32,13 +32,14 @@ message BarrierCompleteResponse {
   string request_id = 1;
   common.Status status = 2;
   repeated CreateMviewProgress create_mview_progress = 3;
-  message GroupedSstableInfo {
+  message LocalSstableInfo {
```

Comment:
Since no longer SSTs to commit are no longer grouped, I make the pb type name consistent with the corresponding type in mem.